util/marketplace.py

```diff
@@ -285,6 +313,8 @@ def __init__(self):
         self.subscription_created = False
 
     def lookup_subscription(self, customer_id, sku_id):
+        if customer_id == TEST_USER["account_number"] and sku_id == "MW02701":
```

Comment:
MW02701 is hardcoded to `RH_SKUS`

Reply:
I can add to the `PLANS` list so that it doesn't have to be hardcoded, but was thinking I would do that in a separate PR so it doesn't clutter this one

Reply:
I meant `MW02701` is already defined in `data/billing.py`. You could just import that variable instead of setting `MW02701` in multiple places